docs/setuptools.txt

```diff
@@ -130,29 +130,29 @@ dependencies, and perhaps some data files and scripts::
 
     from setuptools import setup, find_packages
     setup(
-        name = "HelloWorld",
-        version = "0.1",
-        packages = find_packages(),
-        scripts = ['say_hello.py'],
+        name="HelloWorld",
+        version="0.1",
+        packages=find_packages(),
+        scripts=['say_hello.py'],
 
         # Project uses reStructuredText, so ensure that the docutils get
         # installed or upgraded on the target machine
-        install_requires = ['docutils>=0.3'],
+        install_requires=['docutils>=0.3'],
 
-        package_data = {
+        package_data={
             # If any package contains *.txt or *.rst files, include them:
             '': ['*.txt', '*.rst'],
             # And include any *.msg files found in the 'hello' package, too:
             'hello': ['*.msg'],
         },
 
         # metadata for upload to PyPI
-        author = "Me",
-        author_email = "me@example.com",
-        description = "This is an Example Package",
-        license = "PSF",
-        keywords = "hello world example examples",
-        url = "http://example.com/HelloWorld/",   # project home page, if any
+        author="Me",
+        author_email="me@example.com",
+        description="This is an Example Package",
+        license="PSF",
+        keywords="hello world example examples",
```

Comment:
**Side Note:** Must this really be a blank-separated list of word? Or can commas also be used? I've also seen some developers making a list out if this -- Which is the correct way?